javascript/javascript1/week1/script.js

```diff
@@ -0,0 +1,100 @@
+// Age-ify (A future age calculator)
+const yearOfBirth = 1990;
+const yearFuture = 2030;
+const age = yearFuture - yearOfBirth;
+console.log("You will be " + age + " years old in " + yearFuture);
+
+// Goodboy-Oldboy (A dog age calculator)
+const dogYearOfBirth = 2017;
+const dogYearFuture = 2030;
+let dogYear = dogYearFuture - dogYearOfBirth;
+let typeOfYears = "human"; // Default type of years set to "human"
+
+// Determine whether to show the result in dog years or human years
+const shouldShowResultInDogYears = true; //change this to false if you want to see results in human years
+if (shouldShowResultInDogYears) {
+    dogYear *= 7;
+    typeOfYears = "dog";
+}
+console.log(
+    "Your dog will be " +
+        dogYear +
+        " " +
+        typeOfYears +
+        " years old in " +
+        dogYearFuture
+);
+
+//Housey pricey (A house price estimator)
+
+// Array containing information about friends houses
+const friendsHouseInformation = [
+    {
+        name: "Peter",
+        houseWide: 8,
+        houseDeep: 10,
+        houseHigh: 10,
+        gardenSizeInM2: 100,
+        houseCost: 2500000,
+    },
+    {
+        name: "Julia",
+        houseWide: 5,
+        houseDeep: 11,
+        houseHigh: 8,
+        gardenSizeInM2: 70,
+        houseCost: 1000000,
+    },
+];
+let offer = "too much"; // Default offer set to "too much"
+
+// Loop through each friend's house information
+for (const i of friendsHouseInformation) {
```

Comment:
If we are iterating over an array using the `for ... of` [syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of), we usually name the variables differently. Usually the `i` stand for `index`, but this way we are getting the values here and not the indexes.
So simply you can just rename this to 
```suggestion
for (const value of friendsHouseInformation) {
```
But this means you will have to change all of the instances of `i.houseWide` to `value.houseWide` and `i.houseDeep` to `value.houseDeep`, etc.

Reply:
Thank you for the guidance; I'll definitely keep this point in mind.